### PR TITLE
Image UI for CaseEditor

### PR DESCRIPTION
**Pull request by andreadelrio**
To cover issue #165

### CODE REVIEW

File: src/components/CaseEditor/CaseEditor.js [hunk 1]
```diff
@@ -67,15 +67,42 @@ class _CaseEditor extends Component {
...
     this.setState({ lead: src });
   }
 
+  handleNewImg(img) {
+    this.setState({ newImg: true });
+    let currentImgs = this.props.case.other_images.length
+    this.props.case.other_images[currentImgs] = {url:img};
+  }
+
+  deleteImg(photo) {
```
Comment (davidascher) [inline src/components/CaseEditor/CaseEditor.js]: If I go to a case with one image, add another image, and then delete the first image, then the new image isn't marked as 'lead'.  I think we should always ensure that if there's an image, there's a lead image.

File: src/components/CaseEditor/CaseEditor.scss [hunk 2]
```diff
@@ -52,8 +89,13 @@
...
       &.lead {
         border: 4px solid $interactive-color;
+        .trash {
+          bottom: 1px;
+          border: none;
+        }
         .checkbox {
-          background-image: url("../../img/check.png");
+          border: none;
+          background-image: url("../../img/star.png");
```
Comment (davidascher) [inline src/components/CaseEditor/CaseEditor.scss]: I think we're better off using SVG icons so we have scalability, etc..  See https://github.com/participedia/frontend/blob/448ae34b338bb43dfc16a33ad12b7c914bca7963/src/Upload.js#L78 for an example.

File: src/components/CaseEditor/CaseEditor.scss [hunk 3]
```diff
@@ -41,6 +47,37 @@
...
+        border: 1px solid black;
+        z-index: 1;
+      }
+      .trash {
+        position: absolute;
+        width: 1.4rem;
+        height: 1.4rem;
+        bottom: 6px;
+        right:0;
+        background-image: url("../../img/trash.png");
```
Comment (davidascher) [inline src/components/CaseEditor/CaseEditor.scss]: see point about SVG icons below.

**Comment by andreadelrio** [comment]
@davidascher Added changes to cover edge case.

**Pull request merged**